File: src/main/java/com/yoghurt1131/weatherapi/domain/input/valueobject/RangedWeatherProperty.java
package com.yoghurt1131.weatherapi.domain.input.valueobject;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
public class RangedWeatherProperty {

    @JsonProperty("temp")
    private float temperature;

    @JsonProperty("temp_min")
    private float tempratureMin;

    @JsonProperty("temp_max")
    private float tempratureMax;

    @JsonProperty("pressure")
    private float pressure;

    @JsonProperty("sea_level")
    private float pressureOfSeaLevel;

    @JsonProperty("gnd_level")
    private float pressureOfGroundLevel;

    @JsonProperty("humidity")
    private int humidity;

}

File: src/main/kotlin/dev/yoghurt1131/weatherapi/domain/input/valueobject/RangedWeatherProperty.kt
package dev.yoghurt1131.weatherapi.domain.input.valueobject

import com.fasterxml.jackson.annotation.JsonProperty

data class RangedWeatherProperty(
        @JsonProperty("temp") val temperature: Double,
        @JsonProperty("temp_min") val temperatureMin: Double,
        @JsonProperty("temp_max") val temperatureMax: Double,
        @JsonProperty("pressure") val pressure: Double,
        @JsonProperty("sea_level") val pressureOfSeaLevel: Double,
        @JsonProperty("gnd_level") val pressureOfGroundLevel: Double,
        @JsonProperty("humidity") val humidity: Double
)